samples/native_module_host/src/remote_main.c

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#include <stdio.h>
+#include "proxy_gateway.h"
+#include "module.h"
```

Comment:
Fixed in the latest commit, the file now references "native_module_host.h" so you have some sort of clue where the Module_GetApi() is coming from.